src/features/Folding.ts

```diff
@@ -354,7 +354,7 @@ export class FoldingProvider implements vscode.FoldingRangeProvider {
     ): ILineNumberRangeList {
         const result = [];
 
-        const emptyLine = /^[\s]+$/;
+        const emptyLine = /^[\s]*$/;
```

Comment:
Does JS really require the `[]` here?  Wouldn't `/^\s*$/` be sufficient?

Reply:
Huh....Good point.  Will test

Reply:
Fixed